_docs/welcome/feature-stages.md

```diff
@@ -98,22 +84,4 @@ Below is our list of existing features and their current phases. This informatio
 | [Istio mTLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
 | [Pluggable Key/Cert Support for Istio CA](https://istio.io/docs/tasks/security/plugin-ca-cert.html)        | Beta
 
-### User Interface
```

Comment:
We should keep this section

Reply:
+1, we want to keep these.

Reply:
For UI, i think it is better to call it Observability and have Zipkin and Jaeger in there too.